packages/store/package.json

```diff
@@ -43,7 +43,7 @@
         "jest-extended": "0.11.5",
         "jest-mock-extended": "2.0.4",
         "memfs": "3.3.0",
-        "typescript": "4.0.8",
+        "typescript": "4.3.5",
```

Comment:
I was curious why `store` needed it, and checked. It is required because of `jest-mock-extended` but turns out, we are not using it anymore, therefore, could you please remove `jest-mock-extended` and `typescript` from the `store/package.json`

Reply:
done in https://github.com/SAP/open-ux-tools/pull/671/commits/5cb58366cfe7d6067ac6dc87cfaa78e9e670bd6f and https://github.com/SAP/open-ux-tools/pull/671/commits/34ad550116e139678a99ef3c2f8c621afa14affa